_data/lang/en.yaml

```
@@ -26,7 +26,7 @@ en:
     get_account: Get an Account
     get_account_explanation: You can use (almost) any email account with Delta Chat. If you need a new one, tap or scan this QR code to get a random <a href="https://nine.testrun.org" target="_blank">@nine.testrun.org</a> email address
     add_friends: Enter a Username and Meet Your Friends
-    add_friends_explanation: Now you can enter a username, select a profile picture, and chat with your friends. The best way is to meet them and verify them by <a href="help#howtoe2ee">scanning QR codes</a>.
+    add_friends_explanation: Now you can enter a username, select a profile picture, and chat with your friends. The best way is to <a href="../en/2024-03-12-jumbo44#sharing-invite-links-for-chats-and-chat-groups">send them an invite link</a> or meet them and verify them by <a href="help#howtoe2ee">scanning QR codes</a>.
```

lots of "them" and a long sentence. maybe the follwoing is a bit better?

```suggestion
    add_friends_explanation: Now you can enter a username, select a profile picture, and chat with your friends. The best way is to <a href="../en/2024-03-12-jumbo44#sharing-invite-links-for-chats-and-chat-groups">send them an invite link</a>. Or meet and verify them by <a href="help#howtoe2ee">scanning QR codes</a>.
```